overlord/snapstate/backend_test.go

```diff
@@ -531,22 +544,26 @@ func (f *fakeSnappyBackend) OpenSnapFile(snapFilePath string, si *snap.SideInfo)
 	}
 
 	name := filepath.Base(snapFilePath)
-	if idx := strings.IndexByte(name, '_'); idx > -1 {
-		name = name[:idx]
+	split := strings.Split(name, "_")
+	if len(split) >= 2 {
+		// <snap>_<rev>.snap
+		// <snap>_<instance-key>_<rev>.snap
+		name = split[0]
```

Comment:
Looks like the logic of `SplitInstanceName` helper, can you use it instead?

Reply:
I'm afraid it might be a bit confusing as the input is not an instance name but a file name.

Reply:
I see, thanks.